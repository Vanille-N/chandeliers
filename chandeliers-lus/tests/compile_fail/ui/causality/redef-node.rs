//! Duplicate node definitions.
chandeliers_lus::decl! {
    node foo() returns ();
    let tel;

    node foo() returns ();
    let tel;
}

fn main() {}
