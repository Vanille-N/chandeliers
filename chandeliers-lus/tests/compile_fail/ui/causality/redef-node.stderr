error: Attempt to redefine Node `foo`, when a prior item already defines it
 --> tests/fail/ui/causality/redef-node.rs:5:5
  |
5 | /     node foo() returns ();
6 | |     let tel;
  | |___________^
  |
note: Already defined here
 --> tests/fail/ui/causality/redef-node.rs:2:5
  |
2 | /     node foo() returns ();
3 | |     let tel;
  | |___________^
